src/server/scripts/Outland/BlackTemple/illidari_council.cpp

```diff
@@ -337,11 +323,13 @@ class boss_gathios_the_shatterer : public CreatureScript
                     events.ScheduleEvent(EVENT_SPELL_AURA, 30000);
                     break;
                 case EVENT_SPELL_HAMMER_OF_JUSTICE:
-                    if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, HammerOfJusticeSelector(me)))
-                    {
-                        me->CastSpell(target, SPELL_HAMMER_OF_JUSTICE, false);
-                        events.ScheduleEvent(EVENT_SPELL_HAMMER_OF_JUSTICE, 20000);
-                        break;
+                    if (Unit* target = me->GetVictim()) {
```

Comment:
newline for curly, or delete it entirely as there is an if statement right after (I would do the latter)